app/javascript/components/widgets/widgets/land-cover/global-land-cover/initial-state.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  title: {
+    withLocation: 'Land cover for {location}'
+  },
+  config: {
+    size: 'small',
+    categories: ['land-cover'],
+    admins: ['country', 'region'],
+    selectors: [],
+    years: [2015],
```

Comment:
If no selectors is this needed?

Reply:
Yes it's needed because we're using it to build the dynamic sentence.
AJ wrote in the notebook that we'll need to use a years selector, but is WIP now on the API side.

Reply:
Ok good to know. Thanks!